include/context.h

```diff
@@ -27,7 +27,7 @@ typedef struct {
   VteTerminal* vte;
   GtkBox* hbox;
   GtkBox* vbox;
-  int* uri_tag;
+  int uri_tag;
```

Comment:
https://github.com/endaaman/tym/blob/45bcdcccb1506cf1100b4274550c64ecae1aabb0/src/app.c#L112

This substitution should be also changed.